manifests/m/ManifoldSoftware/Manifold-Release-9/9.0.175/ManifoldSoftware.Manifold-Release-9.yaml

```diff
@@ -0,0 +1,26 @@
+PackageIdentifier: ManifoldSoftware.Manifold-Release-9
+PackageVersion: 9.0.175
+PackageName: Manifold Release 9
```

Comment:
![image](https://user-images.githubusercontent.com/15158490/130680393-97eadf7c-9fac-464a-a3e1-a882486c6170.png)

```suggestion
PackageName: Manifold System
```

Reply:
"Manifold Release 8" and "Manifold Release 9" This is how Manifold Software calls their products. I'd hesitate to remove "Release" from the name.   https://manifold.net/info/manifold8.shtml  
https://manifold.net/info/manifold9.shtml

Reply:
Ok, executable calls product name "Manifold System 9.0".   
`PackageName: Manifold System 9`

Reply:
We can accept this, i would recommend adding the `ProductCode` since i believe WinGet looks for `Manifold Systems 9 *` instead of `Manifold Systems 9*`
I would also suggest changing the PackageIdentifier to `ManifoldSoftware.Systems.8` and `ManifoldSoftware.Systems.9`

Folder structure would then be 
```
- ManifoldSoftware
-- Systems
--- 8
---- Version Folder
----- .Yaml's
--- 9
---- Version folder
----- .Yaml's
```